index.html

```diff
@@ -47,7 +47,7 @@ <h1>I'm Khalil <br> Glad to see you!</h1>
         </section>
         <section class="work" id="my-project">
             <div class="container">
-                <img class="snapshoot-img" src="./Media/Snapshoot-Portfolio-1.png" alt="snapshout portfolio">
+                <img class="snapshoot-img" src="./Media/Snapshoot-Portfolio-1.png" alt="Screenshot of the Tonic project">
```

Comment:
Changed and added more detailed descriptions of the images in the alternative attributes.